src/content/docs/apis/nerdgraph/examples/nerdgraph-entities-api-tutorial.mdx

```diff
@@ -10,28 +10,27 @@ redirects:
   - /docs/apis/graphql-api/tutorials/use-new-relic-graphql-api-query-entities
   - /docs/apis/graphql-api/tutorials/nerdgraph-graphiql-entities-api-tutorial
   - /docs/apis/nerdgraph/tutorials/nerdgraph-graphiql-entities-api-tutorial
+  - /docs/apis/nerdgraph/examples/nerdgraph-relationships-api-tutorial
 ---
 
-With [NerdGraph](/docs/introduction-new-relic-graphql), you can query details about your monitored entities.
+[Entities](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic) at New Relic refer to anything we monitor that generates telemetry. Entities help you find the data you want to track in New Relic, and if you understand their relationships with other entities, you can get even more insights into your data. Here are a few examples of entities:
```

Comment:
maybe 'generates or contains telemetry data'. This helps accounts for custom dashboards and workloads being considered 'entities'.

Reply:
@zuluecho9 I add "...generates or contains telemetry." Thanks!